### PR TITLE
[NB-3832] add notebooks python GPU env

### PR DESCRIPTION
**Pull request by dmytrokarpovych**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Add new python notebook env with GPU support

### CODE REVIEW

File: public_dropin_notebook_environments/python39_notebook_gpu/agent/cgroup_watchers.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    def memory_limit_in_bytes(self) -> float:
+        cgroup_mem_limit = self._cgroup_file_reader.memory_limit_in_bytes()
+        total_virtual_memory = self._system_watcher.virtual_memory().total
+        return min(cgroup_mem_limit, total_virtual_memory)
+
+    def memory_usage_percentage(self) -> float:
+        return round(self.memory_usage_in_bytes() / self.memory_limit_in_bytes() * 100, 2)
+
+    def cpu_usage_limit_in_cores(self) -> float:
```
Comment (klichukb) [inline public_dropin_notebook_environments/python39_notebook_gpu/agent/cgroup_watchers.py:122]: Where is cpu/mem usage going to be used/exposed?

**Review by klichukb — APPROVED** [review]
How heavy is the built env image?

File: public_dropin_notebook_environments/python39_notebook_gpu/agent/cgroup_watchers.py (hunk 1 above)
Comment (dmytrokarpovych) [reply public_dropin_notebook_environments/python39_notebook_gpu/agent/cgroup_watchers.py:122]: It’s the same code that we have for regular images

**Comment by dmytrokarpovych** [comment]
> How heavy is the built env image?

to be precise, it's 9.39GB 🙈

**Comment by cdevent** [comment]
jenkins run test_integration_r_environment test_integration_java_environment

**Pull request merged**